jobs/worker.js

```diff
@@ -45,7 +28,8 @@ const register = () => {
 	app.set('port', parseInt(process.env.ARENA_PORT) || 4000);
 
 	app.use(arenaAuth);
```

Comment:
need `arenaAuth` still?

Reply:
yes, although I could rename it

Reply:
I'd have to update all the env variables as well.  Gonna leave that for another pass.